django_switchuser/state.py

```diff
@@ -11,6 +11,8 @@ def __init__(self, request):
         self._reset()
 
     def _reset(self):
+        User = get_user_model()
```

Comment:
Is there a reason to `get_user_model()` here? Something going on under the hood?